specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimapis.json

```diff
@@ -4415,6 +4415,332 @@
         },
         "x-ms-odata": "./definitions.json#/definitions/TagResourceContract"
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/wikis/default": {
+      "head": {
```

Comment:
>head [](http://example.com/codeflow?start=7&length=4)

I dont think ARM supports the HEAD method. Why do you need this for ?

Reply:
We have HEAD method for most of our specifications. It is used to set the ETag header.

Reply:
Looks like I was mistaken , there are a couple of other RPs using HEAD as well. Thanks.